commitlint/helpers.ts

```diff
@@ -19,16 +19,10 @@ export abstract class Helpers {
     public static convertAnyToString(
         potentialString: any,
         paramName: string
-    ): string {
+    ): any {
```

Comment:
@realmarv I didn't request this change in this PR

Reply:
One of the default-revert-message rule inputs is body, so I needed bodyStr in the plugin and I had to do it. Unless I use rawStr workaround for extracting header and body in this PR. What should I do?

Reply:
> Unless I use rawStr workaround for extracting headerStr and bodyStr in this PR. What should I do?

If the 17.4.0 PR is not merged yet, then you HAVE to use that workaround.